### PR TITLE
健康檢測 API

### PR DESCRIPTION
**Pull request by mikaeloduh**
## Why need this change? / Root cause: 
- Health check API 

## Changes made:
- Adds `GET /health` API

## Test Scope / Change impact:
- `HealthCheckController`

## Issue
- https://github.com/Game-as-a-Service/Lobby-Platform-Service/issues/66

### CODE REVIEW

File: spring/src/test/kotlin/tw/waterballsa/gaas/spring/it/controllers/HealthCheckControllerTest.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class HealthCheckControllerTest {
```
Comment (frankvicky) [inline spring/src/test/kotlin/tw/waterballsa/gaas/spring/it/controllers/HealthCheckControllerTest.kt:13]: 繼承 `AbstractSpringBootTest` ，具體可以參考其他測試。

**Review by frankvicky — CHANGES_REQUESTED** [review]

File: spring/src/test/kotlin/tw/waterballsa/gaas/spring/it/controllers/HealthCheckControllerTest.kt (hunk 1 above)
Comment (mikaeloduh) [reply spring/src/test/kotlin/tw/waterballsa/gaas/spring/it/controllers/HealthCheckControllerTest.kt]: 已修改在 https://github.com/Game-as-a-Service/Lobby-Platform-Service/pull/81/commits/b16a7594185207ac4c3cf9b0702f6872c8dd6332

File: spring/src/main/kotlin/tw/waterballsa/gaas/spring/controllers/HealthCheckController.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class HealthCheckController {
+    @GetMapping("/health")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```
Comment (Wally5077) [inline spring/src/main/kotlin/tw/waterballsa/gaas/spring/controllers/HealthCheckController.kt:11]: `static import`

**Review by Wally5077 — CHANGES_REQUESTED** [review]
Review done.

File: spring/src/main/kotlin/tw/waterballsa/gaas/spring/controllers/HealthCheckController.kt (hunk 2 above)
Comment (mikaeloduh) [reply spring/src/main/kotlin/tw/waterballsa/gaas/spring/controllers/HealthCheckController.kt]: 已修改在 https://github.com/Game-as-a-Service/Lobby-Platform-Service/pull/81/commits/55d55a9176002ebf42015faeb3cd172396639258

**Review by frankvicky — APPROVED** [review]
LGTM

**Review by Wally5077 — APPROVED** [review]
LGTM

**Pull request merged**